packages/purpose-process/test/suspendPurposeVersion.test.ts

```diff
@@ -571,4 +870,159 @@ describe("suspendPurposeVersion", () => {
       );
     }
   );
+  it("should throw organizationNotAllowed when the requester is the Consumer and is suspend a purpose version created by the delegate", async () => {
+    const mockEService = getMockEService();
+    const mockPurposeVersion: PurposeVersion = {
+      ...getMockPurposeVersion(),
+      state: purposeVersionState.active,
+    };
+    const mockPurpose: Purpose = {
+      ...getMockPurpose(),
+      eserviceId: mockEService.id,
+      versions: [mockPurposeVersion],
+    };
+    await addOnePurpose(mockPurpose);
+    await addOneEService(mockEService);
+
+    const delegation = getMockDelegation({
+      kind: delegationKind.delegatedConsumer,
+      eserviceId: mockEService.id,
+      delegateId: generateId<TenantId>(),
+      state: delegationState.active,
```

Comment:
Missing delegationId linking to the purpose, and missing also delegator
```suggestion
    const delegation = getMockDelegation({
      id: mockPurpose.delegationId,
      delegatorId: mockPurpose.consumerId,
      kind: delegationKind.delegatedConsumer,
      eserviceId: mockEService.id,
      delegateId: generateId<TenantId>(),
      state: delegationState.active,
```